File: P181902.py
def solution(my_string):
    answer = [0] * 52
    for digit in my_string:
        if ord('A') <= ord(digit) <= ord('Z'):
            answer[ord(digit)-ord('A')] += 1
        elif ord('a') <= ord(digit) <= ord('z'):
            answer[26+ord(digit)-ord('a')] += 1
    return answer

File: P181902.java
public class P181902 {
    public int[] solution(String my_string) {
        int[] answer = new int[52];
        for (int i=0; i<my_string.length(); i++) {
            char s = my_string.charAt(i);
            if ('A' <= s && s <= 'Z') answer[(int) s - 'A'] += 1;
            else answer[(int) s - 'a' + 26] += 1;
        }
        return answer;
    }
}
